docs/concepts/mode.md

```diff
@@ -27,14 +27,26 @@ Running Santa in Lockdown Mode will stop all blocked binaries and additionally
 will prevent all unknown binaries from running. This means that if the binary
 has no rules or scopes that apply, then it will be blocked.
 
-##### Standalone mode
+##### Standalone mode (Beta)
 
-When Santa is in Standalone Mode it will allow the user to approve their own binaries provided they authenticate biometrically with Touch ID. Upon a successful authentication Santa will then add a `SigningID` rule for the binary if it is validly signed and a `BINARY` if it is not signed at all.
+When Santa is in Standalone Mode it will allow the user to approve their own
+binaries provided they authenticate biometrically with TouchID. Upon a
+successful authentication Santa will then add a `SIGNINGID` rule for the binary
+if it is validly signed and a `BINARY` if it is not signed at all.
 
-When paired with Lockdown, it allows a user to quickly self approve in lieu of using a sync service. If one is using a sync service Events will still be sent up to that sync service.
+This allows a user to quickly self approve in lieu of using a sync service. If
+one is using a sync service Events will still be sent up to that sync service.
+
+Standalone mode typically requires TouchID for self-approval because this
+requires a physical interaction and cannot be spoofed by other software acting
+as a mouse/keyboard. Unfortunately this makes Standalone mode unusable for users
+on most desktop machines or using a laptop in clamshell mode. If supporting that
+use-case is important there is a configuration option
+`EnableStandalonePasswordFallback` that can be enabled to allow falling back to
+password authentication when TouchID is not available.
```

Comment:
```suggestion
Standalone mode typically requires TouchID  or password to approve binary execution.

The configuration option `EnableStandalonePasswordFallback` can be disabled to make Standalone mode require physical access. When disabled only TouchID will be used for self-approval because this requires a physical interaction and cannot be spoofed by other software acting as a mouse/keyboard.
```